### ISSUE TITLE
Add an argument for wait times between requests

### ISSUE THREAD

**Issue by khannover** [arc298/instagram-scraper#725]
**Is your feature request related to a problem? Please describe.**
I would like to scrap some accounts including comments and post metadata but i run into request rate limits and instagram-scraper is throwing errors that requests fail with this messages and i have to interact:

WARNING: Retry after exception HTTPError('429 Client Error: - for url: https://www.instagram.com/graphql/query/?query_hash=[...redacted...]&variables=%7B%22shortcode%22:%22[...redacted...]%22,%22first%22:50,%22after%22:%22%22%7D') on https://www.instagram.com/graphql/query/?query_hash=[...redacted...]&variables={"shortcode":"[...redacted...]","first":50,"after":""}

Repeated error HTTPError('429 Client Error: - for url: https://www.instagram.com/graphql/query/?query_hash=[...redacted...]&variables=%7B%22shortcode%22:%22[...redacted...]%22,%22first%22:50,%22after%22:%22%22%7D')
(A)bort, (I)gnore, (R)etry or retry (F)orever?

**Describe the solution you'd like**
I would like to have a cli argument to specify wait time in seconds between requests. i hope to circumvent the rate limits by instagram if the requests are more spread over the time.

**Describe alternatives you've considered**
I used to use "yes i | instagram-scraper ..." to prevent the interactive inputs but i keep running into timeout wait times.
e feature request here.

**Comment by devanshtyagi150** [comment]
yep i am having the same problem now, have to wait at least 10-15 mins between each scrape to avoid getting these errors

**Comment by Sidiusz** [comment]
Yeah, same for videos
